crates/re_ui/src/command.rs

```diff
@@ -86,6 +87,8 @@ impl UICommand {
             #[cfg(not(target_arch = "wasm32"))]
             UICommand::Open => ("Open…", "Open a Rerun Data File (.rrd)"),
 
+            UICommand::CloseCurrentRecording => ("Close recording", "Close the current Recording"),
```

Comment:
@nikolausWest "dynamic" tooltips would require a significant refactor of that area of code I believe. I'd rather avoid that can of worm in this PR so I'll open an issue. Also, this might be a little-used feature anyways once I add the "delete" button in the recording list.